engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -277,27 +277,29 @@ <h4 translate>Sync Settings</h4>
             on-save="save(repository)">
         </dd>
       </span>
-      <dt translate>SSL CA Cert</dt>
-      <dd bst-edit-select="repository.ssl_ca_cert.name"
-          readonly="product.redhat ||denied('edit_products', product)"
-          selector="repository.ssl_ca_cert_id"
-          options="certs()"
-          on-save="save(repository)">
-      </dd>
-      <dt translate>SSL Client Cert</dt>
-      <dd bst-edit-select="repository.ssl_client_cert.name"
-          readonly="product.redhat ||denied('edit_products', product)"
-          selector="repository.ssl_client_cert_id"
-          options="certs()"
-          on-save="save(repository)">
-      </dd>
-      <dt translate>SSL Client Key</dt>
-      <dd bst-edit-select="repository.ssl_client_key.name"
+      <span ng-hide="repository.content_type === 'ostree' || repository.content_type === 'docker' || repository.content_type === 'ansible_collection'">
```

Comment:
I think it might make more sense to expose these options at update time rather than hide them at creation time.  the ssl_ca_cert option would be useful if pointing to a non-standard galaxy server.  Pulp supports these options for all types